benchmark/benchmarks.jl

```diff
@@ -28,13 +28,13 @@ end
 const LAYER = Conv((5, 5), 3 => 2)
 
 ## Define Benchmark suite
-suite = BenchmarkGroup()
+SUITE = BenchmarkGroup()
 
 for S in SET_TYPES
     setname = string(S)
     method = TracerSparsityDetector(S)
 
-    J_global = suite["Jacobian"]["Global"]
```

Comment:
Yes, I don't have a meaningful benchmark for Hessians.

I am not sure adding the tiny functions from our test-set would make for a useful and informative benchmark.

Reply:
I'll try to come up with an example for the Hessian that is similar to "multiplication by sparse matrix" for the Jacobian

Reply:
Good idea, we could actually reuse exactly that for the Hessian.

Reply:
not exactly I feel, because the following function is not very interesting in terms of tracers mixing (the second order ones only emerge at the very last step):

```julia
function f(x; l, p)
    n = length(x)
    y = copy(x)
    for _ in 1:l
        A = sprand(n, n, p)
        y = A * x
    end
    return dot(x, y)
end
```
but maybe we can adapt it

Reply:
As mentioned in another comment, let's add this in a follow-up.